polkadot/runtime/parachains/src/assigner_parachains.rs

```diff
@@ -17,10 +17,6 @@
 //! The bulk (parachain slot auction) blockspace assignment provider.
 //! This provider is tightly coupled with the configuration and paras modules.
 
-mod mock_helpers;
```

Comment:
Thanks for catching that. 

I remembered from earlier in my bulk assigner work that there were no `tests` or `mock_helpers` files. But since then those files were added.